CONTRIBUTING.md

```diff
@@ -156,10 +156,18 @@ You can run the website locally to see the output of the changes you made to you
 
 ### With Docker
 
+Linux/MacOS
+
 ```bash
 docker run -p 4000:4000 -v $(pwd):/srv/jekyll -it --rm jekyll/jekyll jekyll serve
```

Comment:
Since we are changing this section of the page, the command I use (MacOS) is 
docker run -p 4000:4000 -v $(pwd):/srv/jekyll -it --rm jekyll/jekyll:3 jekyll serve

I added the :3 at the end of jekyll/jekyll on @smellems suggestion as I was getting error messages without it.

Just a thought that we might want to add that here (or make this change in another PR)?